apps/wallet-mobile/src/components/Checkbox/Checkbox.tsx

```diff
@@ -31,9 +31,9 @@ const useStyles = () => {
 
   const styles = StyleSheet.create({
     container: {
-      flex: 1,
       flexDirection: 'row',
-      alignItems: 'center',
+      alignItems: 'flex-start',
+      justifyContent: 'flex-start',
```

Comment:
please go updating when touching it.
```suggestion
      ...atoms.align_start,
      ...atoms.justify_start,
```